src/db/tables/onerep_scans.ts

```diff
@@ -139,25 +140,46 @@ async function getLatestOnerepScan(
   return scan ?? null;
 }
 
+/*
```

Comment:
@rhelmer I'm a bit wary about merging this code in. 1) We don't know how long this code will be relevant 2) especially when it comes to creating new tables, having data in those tables, I'm not sure what the Postgres restrictions we have, and whether it is easy to delete an entire table if necessary

Reply:
@mansaj I think that these changes could live as long as onerep is used in any way. Once onerep is fully obsolete, then this code would need to be rewritten for the new provider (which @codemist asked that I write a doc about before the end of my internship). For the db changes, I think this table could live, but some column names would need to be changed depending on how the data object looks